lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -27,6 +27,9 @@ describe 'StudentAdmin', ->
     it 'binds to the btn_rescore_entrance_exam on click event', =>
       expect(studentadmin.$btn_rescore_entrance_exam).toHandle 'click'
 
+    it 'binds to the $btn_skip_entrance_exam on click event', =>
```

Comment:
@andy-armstrong, do you have a recommendation in this particular case regarding CoffeeScript?